### PR TITLE
week 18 - noemi

### PR DESCRIPTION
**Pull request by NoemiSalome**

### CODE REVIEW

**Review by honkoan — CHANGES_REQUESTED** [review]
Hi lovely teammate Noemi! 🦊😃 

I can see only an empty array or error messages (error messages are working well 🥳) when I'm trying to see your endpoints. I tried to figure out why it is so, but I couldn't. That's the reason I'm requesting changes!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+- API should have at least 2 routes. Try to push yourself to do more, though!
+- A minimum of one endpoint to return a **collection** of results (array of elements)
+- A minimum of one endpoint to return a **single** result (single element).
+- API should make use of Mongoose models to model your data and use these models to fetch data from the database.
+- API should be RESTful
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-week-18.herokuapp.com/
```
Comment (KarolCislowski) [inline README.md:16]: Short documentation would be nice 👍🏻

File: server.js [hunk 2]
```diff
@@ -1,35 +1,136 @@
...
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+//creating schema with data types
+const bookSchema = new mongoose.Schema({
+  bookID: Number,
```
Comment (KarolCislowski) [inline server.js:18]: Why do you need both bookId and _id?

File: server.js [hunk 3]
```diff
@@ -1,35 +1,136 @@
...
+
+if (authors)  {
+  const booksArray = await Book.find({
+    authors: {
+      $regex: new RegExp(authors, 'i')}
+  })
+  res.json(booksArray)
+} else {
+  res.status(400).json({ error: 'something went wrong - sorry about that', details: error })
+}
```
Comment (KarolCislowski) [inline server.js:120]: Nice filtering 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**